src/AddTodoForm.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+  const AddTodoForm = () => {
+    return (
+        <form>
+            <label htmlFor="todoTitle"> Title </label>
+            <input id="todoTitle"></input>
+            <button> Add </button>
```

Comment:
What type of button is this? Is there a property that can clearly define it?

Reply:
got it! done